tools/go-cli/go-whisk-cli/commands/wsk.go

```diff
@@ -50,8 +50,8 @@ func init() {
 
     WskCmd.PersistentFlags().BoolVarP(&flags.global.verbose, "verbose", "v", false, "verbose output")
     WskCmd.PersistentFlags().BoolVarP(&flags.global.debug, "debug", "d", false, "debug level output")
-    WskCmd.PersistentFlags().StringVarP(&flags.global.auth, "auth", "u", "", "authorization key")
-    WskCmd.PersistentFlags().StringVar(&flags.global.apihost, "apihost", "", "whisk API host")
-    WskCmd.PersistentFlags().StringVar(&flags.global.apiversion, "apiversion", "", "whisk API version")
+    WskCmd.PersistentFlags().StringVarP(&flags.global.auth, "auth", "u", "", "authorization `key`")
```

Comment:
@dubeejw - In this PR I focused on the usage syntax (cobra USE:), not the flag descriptions.  I used `action create` as a sample for the follow-on needed changes to the flag descriptions.  Yes, these need changing, but I proposed we fix the errant usage syntax with this PR, then the more lengthy flag description changes in a follow-on PR